### PR TITLE
Added support for settings of gym `xcargs` parameter for purposes of …

### PR DESCRIPTION
**Pull request by lawmaestro**
…being able to set additional build arguments - https://docs.fastlane.tools/actions/build_app/

### CODE REVIEW

File: Fastfile [hunk 1]
```diff
@@ -151,7 +151,13 @@ def _build_with_gym
   export_method = _get_export_method
   xcconfig_filename = Dir.pwd + '/TAB.release.xcconfig'
   create_xcconfig(filename: xcconfig_filename)
-  gym(configuration: _get_build_config, export_method: export_method, xcconfig: xcconfig_filename)
+
+  gym(
+    configuration: _get_build_config,
+    export_method: export_method,
+    xcconfig: xcconfig_filename,
+    xcargs: ENV['GYM_XCARGS'] || ''
```
Comment (yasinahmed81) [inline Fastfile:159]: Where is `GYM_XCARGS ` being set, I can't see it in the iOS PR.

Comment (yasinahmed81) [reply Fastfile:159]: Found it in ci-libraries  👍

**Review by yasinahmed81 — APPROVED** [review]
LGTM 👍

**Review by sversov — APPROVED** [review]
Looks ok to me.

**Pull request merged**